src/drivers/imx/sai.c

```diff
@@ -31,6 +31,7 @@ static void sai_start(struct dai *dai, int direction)
 {
 	dai_info(dai, "SAI: sai_start");
```

Comment:
There are 2 different changes in this patch:
1. support for 16bit mono which results in changes for sywd and twd
2. support for SAI lanes selection (tx0, rx2)

Please split this patch into 2 and provide a better explanation.